src/analysis.cpp

```diff
@@ -1139,60 +1143,105 @@ void MultipoleMoments::_to_disk() {
 // =============== PolymerShape ===============
 
 void PolymerShape::_to_json(json &j) const {
-    using namespace u8;
-    json &k = j["molecules"];
-    for (int i : ids)
-        k[molecules[i].name] = {
-            {bracket("Rg"), Rg.at(i).avg()},
-            {bracket("Re"), Re.at(i).avg()},
-            {bracket("Rg" + squared), Rg2.at(i).avg()},
-            {bracket("Rg" + squared) + "-" + bracket("Rg") + squared, Rg2.at(i).avg() - std::pow(Rg.at(i).avg(), 2.0)},
-            {bracket("Re" + squared) + "/" + bracket("Rg" + squared), Re2.at(i).avg() / Rg2.at(i).avg()},
-            {rootof + bracket("Rg" + squared), sqrt(Rg2.at(i).avg())},
-            {rootof + bracket("Re" + squared), sqrt(Re2.at(i).avg())},
-            {rootof + bracket("Rgxyz" + squared),
-             {sqrt(Rg2x.at(i).avg()), sqrt(Rg2y.at(i).avg()), sqrt(Rg2z.at(i).avg())}}};
-}
-Point PolymerShape::vectorgyrationRadiusSquared(typename Space::Tgroup &g) const {
-    double sum = 0;
-    Point t, r2(0, 0, 0);
-    for (auto &i : g) {
-        double mw = atoms.at(i.id).mw;
-        t = i.pos - g.cm;
-        spc.geo.boundary(t);
-        r2.x() += mw * t.x() * t.x();
-        r2.y() += mw * t.y() * t.y();
-        r2.z() += mw * t.z() * t.z();
-        sum += mw;
+    if (mean.gyration_radius.cnt > 0) {
+        j = {{"molecule", Faunus::molecules[molid].name},
+             {"⟨s²⟩", mean.gyration_radius_squared.avg()},
+             {"⟨s²⟩-⟨s⟩²", mean.gyration_radius_squared.avg() - std::pow(mean.gyration_radius.avg(), 2)},
```

Comment:
If Rg follows a normal distribution, this would be related to a harmonic force constant. We have previously had ideas represent polymers as spheres with fluctuating radii.